containers/Makefile

```diff
@@ -68,6 +68,11 @@ postgresql-clean:
 concent-api-clean:
 	docker rmi $(IMAGE_PREFIX)concent-api:$(IMAGE_VERSION) --force
 
+build/concent-api/RELEASE-VERSION: build/nginx-proxy/concent-api-assets.tar
+	build/virtualenv/bin/python3                       \
+	"build/virtualenv/src/golem-messages/version.py"
+	mv RELEASE-VERSION "build/concent-api"
```

Comment:
The parameters should be indented with respect to the command.

But you could just keep it on a single line and it would be fine.